src/core/verification.service.ts

```
@@ -150,12 +308,36 @@
+              { message: (e as Error).message }
+            );
+          }
+        }
+
+        return await this.verifyJWTProofForAssertion(proofInput, keyId);
+      }
+
       // Construct the data that was originally signed.
       // This means taking the assertion's content *excluding* the entire proof/verification field.
       const assertionDataToCanonicalize = { ...assertion };
       delete (assertionDataToCanonicalize as Partial<Assertion>).verification; // Remove the whole verification field
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)